wgpu-core/src/command/render.rs

```diff
@@ -729,6 +729,7 @@ impl<'a, A: HalApi> RenderPassInfo<'a, A> {
     ) -> Result<StatefulTrackerSubset, RenderPassErrorInner> {
         profiling::scope!("finish", "RenderPassInfo");
         unsafe {
+            profiling::scope!("end_render_pass");
```

Comment:
there is already a scope for this, why do we need the nested one?

Reply:
We don't, must have left that in while investigating.